.github/workflows/cd-frontend.yml

```diff
@@ -28,9 +30,29 @@ on:
           - prod
 
 jobs:
+  wait-for-checks:
+    runs-on: ubuntu-latest
+    outputs:
+      should_deploy: ${{ steps.check-suite.outputs.should_deploy }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Authenticate GitHub CLI
+        run: echo "${{ github.token }}" | gh auth login --with-token
+
+      - name: Run check-suite script
+        id: check-suite
+        run: |
+          ./.github/workflows/check_suite.sh ${{ github.sha }}
```

Comment:
Right now the script lives in the `.github/workflows` dir because that's the only place it's used, but not happy with that. I'm really not sure where I should put the script. I thought about `/bin`, should I dump it there? Should we have a `make` command or is that overkill?